packages/website/_posts/diva-team-at-solana-hacker-house.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: DIVA team at Solana Hacker House
+description: Sharing with you few lines of experience from the Solana Hacker House event representing DIVA at Solana Hacker House, Bangalore.
+date: 10-14 May 2022
+coverImage: 888whales.png
+coverImageDescription: A collection of whales
```

Comment:
image description doesn't make sense for the post

Reply:
Updated the changes requested